pkg/virt-operator/install-strategy/create_test.go

```diff
@@ -732,4 +739,136 @@ var _ = Describe("Create", func() {
+			Expect(err).ToNot(HaveOccurred(), "Should gnenerate previous strategy object")
+			targetStrategy, err = GenerateCurrentInstallStrategy(config, true, namespace)
+			Expect(err).ToNot(HaveOccurred(), "Should gnenerate target strategy object")
+			if prevCustomScc != nil {
+				prevStrategy.customSCCPrivileges = append(prevStrategy.customSCCPrivileges, prevCustomScc)
+			}
+			if currCustomScc != nil {
+				targetStrategy.customSCCPrivileges = append(targetStrategy.customSCCPrivileges, currCustomScc)
+			}
+		}
+
+		executeTest := func(prevCustomScc *customSCCPrivilegedAccounts,
+			currCustomScc *customSCCPrivilegedAccounts,
+			scc *secv1.SecurityContextConstraints,
+			expectedPatch string,
+			addOnly bool) {
+
+			setupPrependReactor(scc.ObjectMeta.Name, []byte(expectedPatch))
+			stores.SCCCache.Add(scc)
+			err = addOrRemoveSSC(targetStrategy, prevStrategy, virtClient, stores, addOnly)
+			Expect(err).ToNot(HaveOccurred(), "Should successfully update users in relevant security context constraints")
+		}
+
+		BeforeEach(func() {
+			stop = make(chan struct{})
+			ctrl = gomock.NewController(GinkgoT())
+			virtClient = kubecli.NewMockKubevirtClient(ctrl)
+			informers.SCC, _ = testutils.NewFakeInformerFor(&secv1.SecurityContextConstraints{})
+			stores.SCCCache = informers.SCC.GetStore()
+			secClient = &secv1fake.FakeSecurityV1{
+				Fake: &fake.NewSimpleClientset().Fake,
+			}
+			virtClient.EXPECT().SecClient().Return(secClient).AnyTimes()
+		})
+
+		AfterEach(func() {
+			close(stop)
+			ctrl.Finish()
+		})
+
+		It("Should remove users that appear in previous strategy but no longer appear in target strategy", func() {
+			existingCustomSCCPriv := generateCustomSCCpriv("test-scc", []string{"someUser2", "someUser3"})
+			targetCustomSCCPriv := generateCustomSCCpriv("test-scc", []string{})
+			scc := generateSCC("test-scc", []string{"someUser1", "someUser2", "someUser3"})
+			expectedJsonPatch := `[ { "op": "test", "path": "/users", "value": ["someUser1","someUser2","someUser3"] }, { "op": "replace", "path": "/users", "value": ["someUser1"] } ]`
+
+			setupStrategies(existingCustomSCCPriv, targetCustomSCCPriv)
+			executeTest(existingCustomSCCPriv, targetCustomSCCPriv, scc, expectedJsonPatch, false)
+		})
+
+		It("Should add users that appear in target strategy but not appeared in previous strategy", func() {
+			existingCustomSCCPriv := generateCustomSCCpriv("test-scc", []string{})
+			targetCustomSCCPriv := generateCustomSCCpriv("test-scc", []string{"someUser2", "someUser3"})
+			scc := generateSCC("test-scc", []string{"someUser1"})
+			expectedJsonPatch := `[ { "op": "test", "path": "/users", "value": ["someUser1"] }, { "op": "replace", "path": "/users", "value": ["someUser1","someUser2","someUser3"] } ]`
+
+			setupStrategies(existingCustomSCCPriv, targetCustomSCCPriv)
+			executeTest(existingCustomSCCPriv, targetCustomSCCPriv, scc, expectedJsonPatch, false)
```

Comment:
Maybe you should use a `DescribeTable` here, instead of duplicating the tests.

Reply:
@danielBelenky Thank you. Done.